operations/node_management/Reboot_NCNs.md

```diff
@@ -138,6 +138,18 @@ The `kubectl` command is installed.
 
         If there are BGP Peering sessions that are not ESTABLISHED on either switch, refer to [Check BGP Status and Reset Sessions](../network/metallb_bgp/Check_BGP_Status_and_Reset_Sessions.md).
 
+    1. Check for components that are in an failed state in CFS.
+
+        Components can be found by running the following command:
+        ```bash
+         ncn-m001# cray cfs components list --status failed
+        ```
+
+        For any NCN components found, reset the error count to 0.  Each component will also have to be disabled in CFS in order to not immediately trigger configuration.  The components will be re-enabled when they reboot.
```

Comment:
There should only be single spacing between sentences.

Also suggest adding an additional sentence after this paragraph:
```text

**NOTE:** Be sure to replace the `<xname>` in the following command with the xname of the NCN component to be reset and disabled.
```

(people often get on autopilot in the docs, so we try to grab their attention when they cannot simply cut and paste a command as written)